spdx/config.py~

```diff
@@ -32,8 +33,8 @@ class TwoWayDict(dict):
 
 def load_license_list():
     FILE_NAME = os.path.join(os.path.dirname(__file__), 'spdx_licenselist.csv')
-    with open(FILE_NAME, 'r') as file_in:
-        reader = csv.DictReader(file_in)
+    with codecs.open(FILE_NAME, 'r', encoding="utf-8") as file_in:
```

Comment:
FILE_NAME and uppercase names are used for global constants by convention and not scoped variables. Use `file_name` instead.